pkg/strings/strings.go

```diff
@@ -0,0 +1,36 @@
+package strings
+
+import (
+	"bufio"
+	"strings"
+)
+
+func Contains(haystack []string, needle string) bool {
```

Comment:
inconsistent with other naming; also, this does not work like a `FindNeedle`-function.